<launch>
	<arg name="uav_id" default="2"/>
	<arg name="uav_amount" default="2"/>
	<group ns="follower_$(arg uav_id)">
		<param name="uav_id" value="$(arg uav_id)"/>
		<param name="uav_amount" value="$(arg uav_amount)"/>
		<node name="follower" pkg="multi_demo" type="follower_uav.py" output="screen"/>
		<node name="plane_control$(arg uav_id)" pkg="uav_controller" type="vector_fw.py" output="screen"/>
	</group>
</launch>
